src/Integration.Vsix/QuickInfo/SonarLintQuickInfoControllerProvider.cs

```diff
@@ -0,0 +1,42 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Utilities;
+
+namespace SonarLint.VisualStudio.Integration.Vsix
+{
+    [Export(typeof(IIntellisenseControllerProvider))]
+    [Name("ToolTip QuickInfo Controller")]
+    [ContentType("text")]
```

Comment:
Using content type "text" will mean this tooltip provider is registered for a whole range of document types that won't contain daemon issues. This isn't necessarily a problem, although it will have a minor performance impact.
However, I don't think we can be more specific about the content type: JavaScript and C++ files have a more specific content type, but _Vue_ files don't.
The only thing we could do is is return _null_ from the _TryCreate_ method if the document type is not one that we support. I'm not sure it's worth it.